rdtools/normalization.py

```diff
@@ -184,17 +184,16 @@ def sapm_dc_power(pvlib_pvsystem, met_data):
                                    poa_diffuse=total_irradiance['poa_diffuse'],
                                    airmass_absolute=airmass_absolute,
                                    aoi=aoi,
-                                   module=pvlib_pvsystem.module,
-                                   reference_irradiance=1)
+                                   module=pvlib_pvsystem.module)
 
     temp_cell = pvlib_pvsystem\
-        .sapm_celltemp(irrad=total_irradiance['poa_global'],
-                       wind=met_data['Wind Speed'],
-                       temp=met_data['Temperature'])
+        .sapm_celltemp(poa_global=total_irradiance['poa_global'],
```

Comment:
This assumes that `pvlib_pvsystem.temperature_model_parameters` are set, or can be inferred from `pvlib_pvsystem.racking_model` and `pvlib_pvsystem.module_type`. I see that `racking_model` is set, but I didn't see where `module_type` is set (values can be `'glass_polymer'` and `'glass_glass'`, default `'glass_polymer'`). The code should work as is, but to be explicit about the temperature model, I would suggest setting `pvlib_pvsystem.module_type` or setting `pvlib_pvsystem.temperature_model_parameters` if that's not already being done. No promises if the current default behavior will continue after pvlib v0.8.0.

That's the only thing I see.

Reply:
Thanks!  I've updated the docstrings and tests to be explicit about the temperature model.  I appreciate the review.

Reply:
Note these are positional args, not keyword args, so I don't promise to deprecate the names if we decide to change them. In particular, I think `temp_air` is going to change at some point in the near future.

Reply:
Thanks @wholmgren.  `setup.py` does specify `pvlib<0.8.0`, but might as well try to be forward-compatible if possible.

Reply:
If things change in future versions of pvlib, is this a case where we can use conditionals to continue to support 0.7?

Reply:
On a related note: If in the future we change to requiring pvlib>0.8, but that doesn't change the RdTools calculations or API, it seems we don't need a major version. (See [Semantic Versioning FAQ](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api))